MdePkg/MdePkg.dec

```diff
@@ -2420,6 +2420,8 @@
   # previous stage has feature enabled and user wants to disable it.
   # BIT 2 = Page-Based Memory Types (Pbmt). This bit is relevant only if
   # previous stage has feature enabled and user wants to disable it.
+  # BIT 3 = Zkr extension.This bit is relevant only if
+  # previous stage has feature enabled and user wants to disable it.

```

Comment:
So, PCD default value is enable for RNG support.

Reply:
Yes because the idea here is that if HW supports it then default it should remain enabled. This PCD ONLY helps override if you want to disable it.